tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -368,7 +370,7 @@ def eval_fn(topic, m, t):
                 else:
                     print('NO MATCH', verbose_pattern(topic, msg, t))          
 
-                total_bytes += len(serialized_bytes) 
+                total_bytes += len(serialized_bytes)
```

Comment:
Please revert unrelated whitespace changes.

Same below.

Reply:
I really thought I caught all of the EOL changes my editor snuck in. Will address along with the requested tests when I have the cycles.

Reply:
@tappan-at-git Please follow up on this soon if you want this patch to be considered for Noetic.

Reply:
> Please follow up on this soon if you want this patch to be considered for Noetic.

@paulbovbel FYI

Reply:
@ tappan-at-git Friendly ping.